src/common/__tests__/cluster-store.test.ts

```diff
@@ -35,7 +35,8 @@ jest.mock("electron", () => {
     app: {
       getVersion: () => "99.99.99",
       getPath: () => "tmp",
-      getLocale: () => "en"
+      getLocale: () => "en",
+      setLoginItemSettings: jest.fn(),
```

Comment:
Can you please remember why this line needed? I've checked test with/without it, and results are equal.

Reply:
This is required if a `mobx` reaction fires to a change on the userPreferences.openAtLogin. Since that reaction is classified as "fireImmediately" and is in an async function, it could fire at any time.